### DISCUSSION TITLE
Custom title with v1

### DISCUSSION THREAD

**Discussion by crummy** [vikejs/vike#944] [Discussion] [answered]
In `+onRenderHtml.tsx` I get my title like this:
```
  const title =
    config.title || 
    "Default Title"
```
Then, in pages, I define a custom title like this:

```
export default {
  title: "Custom Title",
} satisfies Config;
```
When I run `vite build` I get this error:
```
Error: [vite-plugin-ssr@0.4.131][Wrong Usage] /ssg/renderer/+config.h.ts defines an unknown config 'title'
```
But I think from this text found in examples that I should be able to do this:
```
    /** Title defined statically by /pages/some-page/+title.js (or by `export default { title }` in /pages/some-page/+config.js) */
```
I get a similar error if I try to define it in `+title.ts`:
```
Error: [vite-plugin-ssr@0.4.131][Wrong Usage] /ssg/pages/classes/+title.ts defines an unknown config 'title'
```

**Comment by brillout** [comment, marked as answer]
It isn't a built-in config, you have to define it yourself, see how the example use https://vite-plugin-ssr.com/meta.
